### ISSUE TITLE
Cannot rename a tuple element in a declaration

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#23754]
``` C#
    class Program
    {
        static void Main(string[] args)
        {
            var input = GetInput();

            Console.WriteLine(input.time);
        }

        private static (DateTime time, string path) GetInput()
        {
            return default((DateTime, string));
        }
    }
```

1. Attempt to rename "time" in GetInput and all consumptions to be renamed

**Expected:** To be able to rename
**Actual:**

```
---------------------------
Microsoft Visual Studio
---------------------------
You cannot rename this element.
---------------------------
OK   
---------------------------
```

**Comment by jcouv** [comment]
Dupe of https://github.com/dotnet/roslyn/issues/14115 ?

**Comment by jinujoseph** [comment]
closed as dupe of #14115
